### PR TITLE
Fix a minor broken link

### PR DESCRIPTION
**Pull request by austin-schaefer**
Formatting issue in a link

### CODE REVIEW

**Comment by austin-schaefer** [comment]
@paperclypse : nominating you as reviewer since you reported the issue.

File: src/content/docs/alerts-applied-intelligence/new-relic-alerts/troubleshooting/tag-information-not-showing-entity-infra-alert-condition.mdx [hunk 1]
```diff
@@ -12,7 +12,7 @@ redirects:
 
 ## Problem
 
-\[Custom violation description tags]([https://docs.newrelic.com/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/alert-custom-violation-descriptions#example](https://docs.newrelic.com/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/alert-custom-violation-descriptions#example)) are not being replaced by the values from the event.
+[Custom violation description tags](https://docs.newrelic.com/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/alert-custom-violation-descriptions#example) are not being replaced by the values from the event.
```
Comment (jerelmiller) [inline src/content/docs/alerts-applied-intelligence/new-relic-alerts/troubleshooting/tag-information-not-showing-entity-infra-alert-condition.mdx:15]:
```suggestion
[Custom violation description tags](/docs/alerts-applied-intelligence/new-relic-alerts/alert-conditions/alert-custom-violation-descriptions#example) are not being replaced by the values from the event.
```

[nit] You can omit the site URL and use a relative path here. We do this under the hood, but I like to encourage it where possible.

**Comment by austin-schaefer** [comment]
Good catch @jerelmiller , I've been using relative links on the docs site for years and yet I somehow failed to here! 🙃

**Review by jerelmiller — APPROVED** [review]
Looks good to me!

**Pull request merged**